src/community/geomesa/pom.xml

```diff
@@ -0,0 +1,330 @@
+  <packaging>jar</packaging>
+  <version>2.8-SNAPSHOT</version>  
+  <name>GeoMesa Web Module</name>
+
+    <properties>
+        <geomesa.version>1.0.0-rc.2</geomesa.version>
+        <scala.version>2.10.4</scala.version>
+
+        <!-- environment-dependent versions -->
+        <zookeeper.version>3.4.5</zookeeper.version>
+        <accumulo.version>1.5.1</accumulo.version>
+        <hadoop.version>2.2.0</hadoop.version>
+
+        <!-- logging properties
+             we expect logging jars to be provided -->
+        <!-- accumulo 1.4.3 is incompatible with a 1.5.6+ version of slf4j -->
+        <!-- 1.7.x is for hadoop 2 and accumulo 1.5.x -->
+        <slf4j.version>1.7.5</slf4j.version>
+        <scalalogging.version>1.1.0</scalalogging.version>
+
+        <avro.version>1.7.5</avro.version>
+    </properties>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-core-accumulo1.5</artifactId>
+            <version>${geomesa.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.geotools</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.vividsolutions</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-utils-accumulo1.5</artifactId>
+            <version>${geomesa.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.geotools</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.vividsolutions</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+      <dependency>
+          <groupId>org.locationtech.geomesa</groupId>
+          <artifactId>geomesa-jobs-accumulo1.5</artifactId>
+          <version>${geomesa.version}</version>
+          <exclusions>
+              <exclusion>
+                  <groupId>org.geotools</groupId>
+                  <artifactId>*</artifactId>
+              </exclusion>
+          </exclusions>
+      </dependency>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>${scala.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+          <version>11.0.2</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.4</version>

```

Comment:
dependency management from root pom.xml, or you probably have this as a transitive dependency